utam-core/src/main/java/utam/core/selenium/element/DriverAdapter.java

```diff
@@ -11,16 +11,14 @@
 
 import java.time.Duration;
 import java.util.List;
+import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import org.openqa.selenium.By;
-import org.openqa.selenium.JavascriptExecutor;
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.WebDriver;
+
+import org.openqa.selenium.*;
```

Comment:
Avoid `.*` imports. Yes, the IDE may do this for you, but you'll need to revert that change when it happens.